jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/popup/CodeWhispererUIChangeListener.kt

```diff
@@ -92,5 +94,8 @@ class CodeWhispererUIChangeListener : CodeWhispererPopupStateChangeListener {
 
     override fun recommendationAdded(states: InvocationContext, sessionContext: SessionContext) {
         CodeWhispererPopupManager.getInstance().updatePopupPanel(states, sessionContext)
+        if (!CodeWhispererInvocationStatus.getInstance().hasExistingInvocation()) {
+            CodeWhispererTelemetryService.getInstance().sendClientComponentLatencyEvent(states)
+        }
```

Comment:
So this will only get sent after adding a recommendation _and_ pagination has terminated?